styles.css

```diff
@@ -0,0 +1,53 @@
+body {
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Helvetica Neue',
+    Arial,
+    sans-serif;
+}
+
+main {
+  background-color: #eee;
+}
+
+#title,
+p {
+  text-align: center;
+  padding-top: 20px;
+}
+
+#image {
+  display: block;
+  max-width: 100%;
+  height: auto;
+  margin-left: auto;
+  margin-right: auto;
+  justify-content: center;
+}
+
+#img-caption {
+  text-align: center;
+}
+
+#tribute-info {
+  display: block;
+  width: 70%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+#tribute-info ul li {
+  padding-top: 15px;
+}
+
+#cite {
+  margin: 20px;
+}
+
+#further {
+  text-align: center;
+  padding-bottom: 15px;
```

Comment:
i would suggest that bottom padding should be Normal; Not Very High and Not very Low Like 23px;

Reply:
**1-** Please consider not using the words like `should or ought or something else` as you are not supposed to give orders else you are here to advise the user of a descriptive change with a valid reason for the change. Consider editing your comment to make it in accordance with the above-mentioned instructions 👍

**2-** Please consider always including an explanation, a **"why"**, for each requested change. This means to describe the user why he is being asked for some change.

## General example:-

* **This is a well-explained request:** *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*

* **This request lacks a "why":** *Why don't you use an article tag here instead of a div?*

**3-** It's good to use some `gifs` but consider not using them in the inline comments as it feels distracting else you can use them in the review description. But you can use emojis here 👍